ui/library/less/styles.less

```diff
@@ -38,3 +47,8 @@
 .react-pdf__Page__textContent>span {
   height: 0em!important;
 }
+
+.reader__download-btn {
```

Comment:
Same comments as above, use `button` for `btn`

Reply:
Careful if you update this that the class name on the S2 side gets updated as well.

Reply:
@yensung i dont want to change this to `button` since in S2 side we have a class like that

Reply:
This would also be a good conversation to have when reviewing the CSS prefixes- `btn` or `button`? Decide on a standard then fix and test everything all at once.